block_context.go

```diff
@@ -7,18 +7,29 @@ package slack
 type ContextBlock struct {
 	Type     MessageBlockType `json:"type"`
 	BlockID  string           `json:"block_id,omitempty"`
-	Elements []BlockObject    `json:"elements"`
+	Elements ContextElements  `json:"elements"`
 }
 
-// blockType returns the type of the block
-func (s ContextBlock) blockType() MessageBlockType {
+// BlockType returns the type of the block
+func (s ContextBlock) BlockType() MessageBlockType {
 	return s.Type
 }
 
-// NewContextBlock returns a newinstance of a context block
-func NewContextBlock(blockID string, elements ...BlockObject) *ContextBlock {
+type ContextElements struct {
+	ContextElementSet []MixedElement
```

Comment:
s/ContextElementSet/Elements/

Reply:
agree `Elements` is better here. I went with `BlockSet` in the case of `Blocks` because I couldn't really come up with anything better, but totally open to suggestions

Reply:
Maybe `LayoutBlocks` would be a better naming choice than `BlockSet`?